src/styles/layouts/_container.scss

```diff
@@ -0,0 +1,6 @@
+.l-container {
+	max-width: 1216px;
```

Comment:
Willen we, in het algemeen, consistente `rem` opleggen als basis unit?

Reply:
I vote yes.